app/assets/v2/js/pages/bounty_details.js

```diff
@@ -575,6 +575,30 @@ var attach_snoozee_options = function() {
   });
 };
 
+var show_notify_funder_modal = function() {
+  var self = this;
+
+  setTimeout(function() {
+    var url = '/notifyfunder/modal?redirect=' + window.location.pathname + '&pk=' + document.result['pk'];
```

Comment:
- Do we need a redirect?
- would it make sense to name it as `/<issue-id>/modal/notify-funder` ?

Reply:
@thelostone-mc I was following the pattern here:
https://github.com/gitcoinco/web/blob/281b8fa76ae75b8b9878e0f79bf84028793862d3/app/assets/v2/js/pages/bounty_details.js#L614-L618
Should it be different from the show interest modal?

Reply:
@oogetyboogety could we stick to this format ? 
we are fixing it up slowly

Reply:
@thelostone-mc Sure, I have updated it now to use the stdbounties_id, which I assume was the <issue-id> you were referring to...I didn't see <issue-id> in the urls.py file